.gitignore

```diff
@@ -1,6 +1,12 @@
 dist
 dist-site
-!tokens/dist
+!tokens/dist/index.css
+!tokens/dist/css/*.css
+!tokens/dist/css/express/*.css
+!tokens/dist/css/spectrum/*.css
```

Comment:
Updated this to target just the assets that can be changed by build updates to tokens since the goal of committing them was to see any breaking changes that might occur. Since the assets generated here are being added to the tokens src section, we have a record of any changes occurring so we don't need to duplicate that by also committing it's dist copy to the repo.